projects/schematics/src/shared/lib-configs/pickup-in-store-schematics-config.ts

```diff
@@ -17,7 +17,7 @@ import {
 import { SchematicConfig } from '../utils/lib-utils';
```

Comment:
for line 58 for `dependencyFeatures` 

I saw this in the history before https://github.com/SAP/spartacus/commit/20e072d40514f8b33f6f26eb0255f1b93a924a3e#diff-c4fd083d326d3f805913413a393689971fa098a77f6778ffe941bf801c510623L34-L39

Are all the listed features in the array for schematics necessary for this feature to function?

```
    CART_BASE_FEATURE_NAME,
    ORDER_FEATURE_NAME,
    STOREFINDER_FEATURE_NAME,
    USER_PROFILE_FEATURE_NAME,
```
Basically, are cart base, order, storefinder, user profile features necessary for this feature to work properly?

Thinking out loud, don't we need checkout for this feature to work properly (in a sense that you confirm for pick up after the confirmation order) --- I'm not too familiar with bopis, so I'm not really sure how the feature works without actually digging.